### ISSUE TITLE
Support null safety

### ISSUE THREAD

**Issue by md84419** [Pyozer/dots_indicator#13]
```
The library 'package:dots_indicator/dots_indicator.dart' is legacy, and should not be imported into a null safe library.
Try migrating the imported library.
```

```
Error: This project cannot run with sound null safety, because one or more project dependencies do not
support null safety:

- dots_indicator
```

```
dependencies:
  dots_indicator: ^1.2.0
```

**Comment by Ethirallan** [comment]
Thanks to @hlavki's PR #15, you can use code below until the PR is merged.

```
  dots_indicator:
    git:
      url: https://github.com/hlavki/dots_indicator.git
      ref: master
```

**Comment by Pyozer** [comment]
Hi @md84419 and @Ethirallan, I just published v2.0.0 with PR #15 merged.
Sorry for not being very present.
